BAZProjectC1/Billboard/View/BillboardViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  BillboardViewController.swift
+//  BAZProjectC1
+//
+//  Created by efloresco on 21/09/22.
+//
+
+import Foundation
+import UIKit
+
+class BillboardViewController : UIViewController {
+    @IBOutlet weak var clvListMovies: UICollectionView!
+    var lstMovies = [Movie]()
+    let columnNum: CGFloat = 3
```

Comment:
Estas propiedades podrían ser privadas?
```suggestion
    private var lstMovies = [Movie]()
    privtae let columnNum: CGFloat = 3
```